asset/api/Asset.Samples.Tests/ExportAssetsTest.cs

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using System;
+using Xunit;
+
+[Collection(nameof(AssetFixture))]
+public class ExportAssetsTest
+{
+    private readonly AssetFixture _fixture;
+    private readonly ExportAssetsSample _sample;
+
+    public ExportAssetsTest(AssetFixture fixture)
+    {
+        _fixture = fixture;
+        _sample = new ExportAssetsSample();
+    }
+
+    [Fact]
+    public void TestExportAssets()
+    {
+        // Run the sample code.
+        var result = _sample.ExportAssets(_fixture.BucketName, _fixture.ProjectId);
+
+        string expectedOutput = String.Format("\"outputConfig\": {{ \"gcsDestination\": {{ \"uri\": \"gs://{0}/my-assets.txt\" }} }}", _fixture.BucketName);
+        Assert.Contains(expectedOutput, result.ToString());

```

Comment:
Can you check the actual field instead of converting this to string. It's really harder to maintaing this code if at some point we want to test for something else.
Here and everywhere where similar code might be present.

Reply:
These RPCs are owned by our sister team (we share the same public api). I'm not familiar with these RPCs and how to meaningfully check the results, so I'd rather not change their existing tests. I can file a bug for them to update the tests.

Reply:
OK, but it's as simple as this:
```suggestion
        Assert.Equals($"gs://{_fixture.BucketName}/my-assets.txt", result.OutputConfig.GcsDestination.Uri);
```
ToString is only generating a JSON like representation of the actual object.
I'm fine if you don't want to change this, but please do create an issue for the other team to do so. This is very fiddly code.